dvc/repo/artifacts.py

```diff
@@ -85,3 +112,203 @@ def add(self, name: str, artifact: Artifact, dvcfile: Optional[str] = None):
             self.repo.scm_context.track_file(dvcfile)
 
         return artifacts.get(name)
+
+    def get_rev(
+        self, name: str, version: Optional[str] = None, stage: Optional[str] = None
+    ):
+        """Return revision containing the given artifact."""
+        from gto.tag import find as find_tags
```

Comment:
Is this the only substantial thing that we are using from?

Reply:
Yes